infra/Makefile

```
@@ -1,4 +1,10 @@
-.PHONY: help migrate-up migrate-down migrate-status docker-up docker-down clean docker-prod-up generate-secrets setup-env migrate-force dev-setup migrate-refresh all test
+.PHONY: help \ 
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)